src/repository/opamDownload.ml

```diff
@@ -132,7 +132,8 @@ let download_command ~compress ?checksum ~url ~dst () =
       OpamConsole.error_and_exit `Configuration_error
         "Empty custom download command"
   in
-  OpamSystem.make_command ~allow_stdin:false cmd args @@> tool_return url
+  let stdout = OpamSystem.temp_file "dl" in
```

Comment:
The file need to be cleaned. It can be done explicitely (see `OpamGit.diff` or `OpamSysInteract.run_command`, 2 different way to do it), or let to the log cleaner to do it. We want to ensure that everything that is created in user temp directory is cleaned by opam, as far as possible.

Reply:
done